content/english/facts/soy-is-bad-for-you.md

```diff
@@ -0,0 +1,191 @@
+
+This was our biggest surprise in researching this topic, as we had heard over and over again that soy contained _plant_ estrogen, you don't need to worry about plant estrogen as they can't interact with the human body hormonally!
+
+Unfortunately phytoestrogens are not so easily dismissed. See *[Soy contains estrogen]({{<ref "/facts/soy-contains-estrogen">}})* for more.
+
+## 2. Dosage is hard to understand
+
+This assertion is based on:
+
+1. How much phytoestrogen is in food is difficult to measure and varies wildly
+2. How much is safe to eat is not well understood
+
+
+### 2.1 Hard to tell how much phytoestrogen is in food
+The amount of phytoestrogens (e.g. isoflavones) in soy based products varies wildly depending on the product, the origin of the soybeans, and the cooking method.  
+
+Even if there were a Recommended Daily Allowance, it would be quite hard to tell whether you were above or below it based on the nutritional information available.  
+
+> In Canada, isoflavones [a type of phytoestrogen] in soy beans vary from 360 to 2241 mg per kg. Isoflavones content in soy beans from Romania ranged between 210 and 1340 g per kg, 1176-3309 mg per kg in USA  and 525-986 mg per kg in India. 
+> - [Soy, Soy Foods and Their Role in Vegetarian Diets - Nutrients - 2018 - p7](https://web.archive.org/web/20191002195620/https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5793271/pdf/nutrients-10-00043.pdf/)
+
+So even at the source the amount of isoflavones per kg can vary massively depending on the local conditions of the crop.
+
+> In soy flour, total isoflavone concentrations range from 60 to 265 mg per 100 g, tofu may contain between 5.1 and 64 mg per 100 g of total isoflavones, soy milk 1.3-21 mg per 100 g, tempeh 6.9-63 mg per 100 g, soy sauce 0.1-2.3 mg per 100 g, miso 23-126 mg per 100 g and natto 20-124 mg per 100 g.  
+> [...]  
+> Temperature is critical for retention of isoflavones: in fact, soymilk film formed during boiling, called Foo joke or Yuba, has been reported to have 196.05 mg per 100 g of isoflavone compared to 44.67 mg per 100 g after cooking   
+> [...]  
+> The most refined soy foods can lose up to 80-90% of isoflavone content during processing. Soy burgers have 0.1-26 mg per 100 g of total aglycone equivalents, soy yogurts 1.6-11.8 mg per 100 g, soy drinks 1.0-21 mg per 100 g and soy cheeses 2.3-33 mg per 100 g. Usually, bakery soy-based products retain low concentration of isoflavones. Soy milk formulas may contain up to 31 mg per 100 g.  
+> - [Soy, Soy Foods and Their Role in Vegetarian Diets - Nutrients - 2018 - p18](https://web.archive.org/web/20191002195620/https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5793271/pdf/nutrients-10-00043.pdf/)
+
+In other words, foods containing soy can have very unpredictable amounts of phytoestrogens, processed foods can lose a lot of their isoflavone content, and usually the effect of cooking soy based foods is to reduce the amount of isoflavones dramatically.
```

Comment:
Just the way I read it. Sort of sounds like cooking is done on purpose to reduce the isoflavones not that cooking just has that effect naturally.

Reply:
Huh, yeah because of the way the sentence scans it's easy to insert "purpose" instead of "effect" when skimming it. I'll rephrase.